### PR TITLE
Smallvil fixes 2

### PR DESCRIPTION
**Pull request by YaLTeR**
This time: fix clipboard and add basic popup support.

### CODE REVIEW

**Review by PolyMeilex — APPROVED** [review]
Looks great 👍

File: smallvil/src/handlers/compositor.rs [hunk 1]
```diff
@@ -40,6 +43,25 @@ impl CompositorHandler for Smallvil {
...
+                    .lock()
+                    .unwrap()
+                    .initial_configure_sent
+            });
+            if !initial_configure_sent {
+                // NOTE: This should never fail as the initial configure is always
+                // allowed.
+                popup.send_configure().expect("initial configure failed");
+            }
+        }
```
Comment (PolyMeilex) [inline smallvil/src/handlers/compositor.rs:64]: Could we move this to xdg specific commit handler? Aka `xdg_shell::handle_commit` from line 44
My idea was to have everything xdg related in a single xdg file, where both commit and xdg handler logic lives

**Review by PolyMeilex — CHANGES_REQUESTED** [review]

File: smallvil/src/handlers/compositor.rs (hunk 1 above)
Comment (YaLTeR) [reply smallvil/src/handlers/compositor.rs]: Done.

**Review by PolyMeilex — APPROVED** [review]
Thanks! Popups support in Smallvil was long overdue, so it's great to see it.

**Pull request merged**